apps/webservice/src/app/[workspaceSlug]/settings/(settings)/account/profile/page.tsx

```diff
@@ -1,15 +1,142 @@
-export const metadata = { title: "Profile" };
+import React from "react";
+import Image from "next/image";
+import { redirect } from "next/navigation";
+import { SiGithub, SiGooglecalendar, SiSlack } from "react-icons/si";
+
+import { auth } from "@ctrlplane/auth";
+import { Button } from "@ctrlplane/ui/button";
+import { Input } from "@ctrlplane/ui/input";
+import { Label } from "@ctrlplane/ui/label";
+
+import { api } from "~/trpc/server";
+import UpsertUsername from "./UpsertUsername";
+
+const defaultAvatar = "/apple-touch-icon.png";
+
+export default async function AccountSettingProfilePage() {
+  const session = await auth();
+  if (!session) redirect("/login");
+  const user = await api.user.viewer();
+  const { image, name, email } = user;
 
-export default function AccountSettingProfilePage() {
   return (
-    <div className="container mx-auto max-w-2xl space-y-8">
-      <div className="space-y-1">
-        <h1 className="text-xl font-semibold">Profile</h1>
-        <p className="text-sm text-muted-foreground">
-          Manage your Ctrlplane profile
-        </p>
+    <div className="scrollbar-thin scrollbar-thumb-neutral-800 scrollbar-track-neutral-900 h-[calc(100vh-120px)] overflow-auto">
+      <div className="container mx-auto max-w-2xl space-y-8 py-8">
+        <div className="space-y-1">
+          <h1 className="text-xl font-semibold">Profile</h1>
+          <p className="text-sm text-muted-foreground">
+            Manage your Ctrlplane profile
+          </p>
+        </div>
+
+        {/* Profile Section */}
+        <ul>
+          <section className="space-y-6 rounded-t-lg bg-neutral-800/70 p-6 shadow-lg">
+            <li>
+              <div className="relative flex items-center justify-center">
+                <Label className="absolute left-0 top-0 text-sm">
+                  Profile picture
+                </Label>
+                <Image
+                  src={image ?? defaultAvatar}
+                  alt="Profile picture"
+                  quality={100}
+                  width={100}
+                  height={100}
+                  className="rounded-full"
+                  priority={true}
+                  draggable={false}
+                />
+              </div>
+            </li>
+          </section>
+          <section className="space-y-6 rounded-b-lg border border-neutral-800/70 bg-background px-6 shadow-lg">
+            {/* Email */}
+            <li className="border-b py-6 lg:mt-0">
+              <div className="flex flex-row items-center justify-between">
+                <Label>Email</Label>
+                <Input readOnly value={email} className="w-64" />
+              </div>
+            </li>
```

Comment:
the label says email??

Reply:
<img width="657" alt="Screenshot 2024-09-19 at 11 04 54 PM" src="https://github.com/user-attachments/assets/f7072b3f-d92b-46cc-95e7-1006a9667b87">

Yes it does.